src/utils/footer.ts

```diff
@@ -0,0 +1,22 @@
+import { FooterLink } from '../types/footer';
+
+export const footerLinks: Array<FooterLink> = [
+  {
+    label: 'All',
+    dataCy: 'FilterLinkAll',
+    href: '#/',
+    key: 'all',
+  },
+  {
+    label: 'Active',
+    dataCy: 'FilterLinkActive',
+    href: '#/active',
+    key: 'active',
+  },
+  {
+    label: 'Completed',
+    dataCy: 'FilterLinkCompleted',
+    href: '#/completed',
+    key: 'completed',
+  },
+];
```

Comment:
Don't need to create array, you can use `Object.values(your created enum)` for creating array